src/core/lv2/Lv2SubPluginFeatures.cpp

```diff
@@ -67,54 +69,128 @@ Lv2SubPluginFeatures::Lv2SubPluginFeatures(Plugin::Type type) :
 
 
 
-void Lv2SubPluginFeatures::fillDescriptionWidget(QWidget *parent,
-													const Key *k) const
+void addHbox(QWidget* parent, const char* left, QString right)
 {
-	const LilvPlugin *plug = getPlugin(*k);
-
-	auto label = new QLabel(parent);
-	label->setText(QWidget::tr("Name: ") + pluginName(plug));
+	if (right.isEmpty()) { return; }
 
-	auto label2 = new QLabel(parent);
-	label2->setText(QWidget::tr("URI: ") +
-		lilv_node_as_uri(lilv_plugin_get_uri(plug)));
-
-	auto maker = new QWidget(parent);
-	auto l = new QHBoxLayout(maker);
+	auto container = new QWidget(parent);
+	auto l = new QHBoxLayout(container);
 	l->setContentsMargins(0, 0, 0, 0);
 	l->setSpacing(0);
 
-	auto maker_label = new QLabel(maker);
-	maker_label->setText(QWidget::tr("Maker: "));
-	maker_label->setAlignment(Qt::AlignTop);
+	auto leftLabel = new QLabel(container);
+	leftLabel->setText(QWidget::tr(left));
+	leftLabel->setAlignment(Qt::AlignTop);
 
-	auto maker_content = new QLabel(maker);
-	maker_content->setText(
-		qStringFromPluginNode(plug, lilv_plugin_get_author_name));
-	maker_content->setWordWrap(true);
+	auto rightLabel = new QLabel(container);
+	if (right.startsWith("http") && !right.contains(' ') && !right.contains('\n'))
+	{
+		right = QString("<a href=\"%1\">%2</a>").arg(right, right);
+		rightLabel->setTextInteractionFlags(rightLabel->textInteractionFlags()
+			| Qt::LinksAccessibleByMouse | Qt::LinksAccessibleByKeyboard);
+		rightLabel->setTextFormat(Qt::RichText);
+		QObject::connect(rightLabel, &QLabel::linkActivated, [](const QString & link){
+			QDesktopServices::openUrl(link);});
+	}
+	rightLabel->setText(right);
+	rightLabel->setWordWrap(true);
 
-	l->addWidget(maker_label);
-	l->addWidget(maker_content, 1);
+	l->addWidget(leftLabel);
+	l->addWidget(rightLabel, 1);
+}
 
-	auto copyright = new QWidget(parent);
-	l = new QHBoxLayout(copyright);
-	l->setContentsMargins(0, 0, 0, 0);
-	l->setSpacing(0);
-	copyright->setMinimumWidth(parent->minimumWidth());
 
-	auto copyright_label = new QLabel(copyright);
-	copyright_label->setText(QWidget::tr("Copyright: "));
-	copyright_label->setAlignment(Qt::AlignTop);
 
-	auto copyright_content = new QLabel(copyright);
-	copyright_content->setText("<unknown>");
-	copyright_content->setWordWrap(true);
-	l->addWidget(copyright_label);
-	l->addWidget(copyright_content, 1);
 
-	AutoLilvNodes extensions(lilv_plugin_get_extension_data(plug));
-	(void)extensions;
-	// possibly TODO: version, project, plugin type, number of channels
+void addLabel(QWidget* parent, const char* left, QString right)
+{
+	auto lbl = new QLabel(parent);
+	if (right.isEmpty()) { return; }
+	if (right.startsWith("http") && !right.contains(' ') && !right.contains('\n'))
+	{
+		right = QString("<a href=\"%1\">%2</a>").arg(right, right);
```

Comment:
Should be able to repeat `%1` for multiple substitutions of the same value:
```suggestion
		right = QString("<a href=\"%1\">%1</a>").arg(right);
```